### PR TITLE
[introspection] Fix bindings so the tests pass

### PR DESCRIPTION
**Pull request by VincentDondain**
- Added missing NSCopying, NSCoding and NSSecureCoding.
- Filled radar://26939747 with Apple - https://trello.com/c/6aIzLH4a

### CODE REVIEW

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by VincentDondain** [comment]
While this is indeed fixing the introspection tests for iOS, it looks like it also broke some mac tests (dontlink-mac, introspection-mac) as well as the mini tests...

**Comment by spouliot** [comment]
* mini tests are not working right now;
* dontlink-mac and introspection-mac are identical (there's another PR out to de-duplicated them)

File: tests/introspection/iOS/iOSApiProtocolTest.cs [hunk 1]
```diff
@@ -154,6 +154,10 @@ protected override bool Skip (Type type, string protocolName)
 				case "AVMutableComposition":
 				case "AVCompositionTrackSegment":
 				case "MKMapSnapshotOptions":
+				case "NSLayoutAnchor`1":
+				case "NSLayoutDimension":
+				case "NSLayoutXAxisAnchor":
+				case "NSLayoutYAxisAnchor":
```
Comment (spouliot) [inline tests/introspection/iOS/iOSApiProtocolTest.cs]: if those types exists in **macOS** then the `case`s must be moved into the shared `ApiProtocolTest.cs` and not the iOS specific file. That might be why you have issues with the mac tests...

Comment (VincentDondain) [inline tests/introspection/iOS/iOSApiProtocolTest.cs]: Indeed those types also exist in `AppKit`, thanks.

However this isn't what's causing the mac tests to fail.

**Comment by xamarin-release-manager** [comment]
Build success

**Comment by VincentDondain** [comment]
Better, the mac tests pass now (:

**Pull request merged**